### PR TITLE
update virtual yogibear params and add readable check to virtual data updates

### PR DESCRIPTION
**Pull request by karthik-shanmugam**
Addressing Issue #166 
Now that YogiBear firmware has been merged the virtual device has to be updated to match its behavior

### CODE REVIEW

File: hibike/virtual_device.py [hunk 1]
```diff
@@ -51,13 +51,17 @@
...
+                         ("pid_pos_kd", 5.0), ("pid_vel_setpoint", 6.0), ("pid_vel_kp", 7.0),
+                         ("pid_vel_ki", 8.0), ("pid_vel_kd", 9.0), ("current_thresh", 10.0),
+                         ("enc_pos", 11.0), ("enc_vel", 12.0), ("motor_current", 13.0)]
         
 while (True):
     if (updateTime != 0 and delay != 0):
         if((time.time() - updateTime) >= (delay * 0.001)):
 #If the time equal to the delay has elapsed since the previous device data, send a device data with the device id and the device's subscribed params and values
-            data = [data_tuple for data_tuple in params_and_values if data_tuple[0] in subscribed_params]
+            data = [data_tuple for data_tuple in params_and_values if data_tuple[0] in subscribed_params and hm.paramMap[device_id][data_tuple[0]][2]]
```
Comment (nikitakit) [inline hibike/virtual_device.py]: helper for determining if param is readable

**Review by nikitakit — CHANGES_REQUESTED** [review]

**Review by nikitakit — APPROVED** [review]
lgtm

**Comment by nikitakit** [comment]
Approved, but please adjust the commit title as you merge.

**Comment by yizow** [comment]
Before merging please make sure title has [HIBIKE] tag

**Pull request merged**